testdata/project-v2/config/default/kustomization.yaml

```diff
@@ -20,18 +20,13 @@ bases:
 #- ../webhook
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
 #- ../certmanager
+# [PROMETHEUS] To enable prometeus-manager, uncomment all sections with 'PROMETHEUS'. 'WEBHOOK' components are required.
+#- ../prometheus
 
 patchesStrategicMerge:
-  # Protect the /metrics endpoint by putting it behind auth.
-  # Only one of manager_auth_proxy_patch.yaml and
-  # manager_prometheus_metrics_patch.yaml should be enabled.
+# Protect the /metrics endpoint by putting it behind auth.
+# Only one of manager_auth_proxy_patch.yaml
```

Comment:
this comment doesn't make sense any more, but also, why are we removing it and this section?  You still might need protection on your prometheus endpoint.